style.css

```diff
@@ -412,3 +448,230 @@ ul .small-button-wrapper {
   margin-bottom: 0;
   font-size: 15px;
 }
+
+@media (min-width: 992px) {
```

Comment:
- [ ] Please, use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px), for example: `@media screen and (min-width: 992px) {` 🐶